lib/assessment.js

```diff
@@ -13,14 +19,25 @@ var min = function min() {
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+  var total = 0;
+  for (var i = 0; i < arguments.length; i++) {
+    total = total + arguments[i];
+  }
+  return total;
 };
 
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
+var defaultArray = [];
 var newArray = function newArray(length, defaultValue) {
-
+var i = 0;
+while (length > 0) {
```

Comment:
By choosing to test for `length > 0`, you have constrained yourself into using `length` as a countdown for this loop. We'd rather not alter our parameters' values, as we may need them later. Instead, consider counting *up* from `i` to `length` by setting the loop condition to `i < length`.